src/apps/backend/modules/logger/internals/grafana-loki-logger.ts

```diff
@@ -1,18 +1,18 @@
 import { format } from 'winston';
 import LokiTransport from 'winston-loki';
 
-import ConfigService from '../../config/config-service';
+import { ConfigService } from '../../config';
 
 import WinstonLogger from './winston-logger';
 
 export default class GrafanaLokiLogger extends WinstonLogger {
```

Comment:
We should consider removing all types except Datadog which I believe is the official stand moving forward.

Reply:
Addressed